src/documents/components/DataValidation/DataValidation.js

```diff
@@ -0,0 +1,388 @@
+                  }}
+                  linkTarget="_blank"
+                />
+              </Segment>
+            ) : (
+              <Segment basic secondary placeholder>
+                <Header icon as="h4" disabled>
+                  <Icon name="clipboard outline" />
+                  No data validation report available
+                </Header>
+              </Segment>
+            )}
+          </Grid.Column>
+        </Grid.Row>
+      ) : (
+        <Grid.Row className="noVerticalPadding">
+          <Grid.Column className="pr-0 pt-20">
+            {hasReport === 'not_started' && (
+              <Segment placeholder>
+                <Header icon>
+                  <Icon name="clipboard check" />
+                  No data validation report created
+                </Header>
+                {allowStartValidation && (
+                  <Button
+                    primary
+                    onClick={() => {
+                      setHasReport('running');
+                      startValidationRun({
+                        variables: {
+                          input: {
+                            dataReview: reviewNode.id,
+                          },
+                        },
+                      })
+                        .then(resp => {
+                          startPolling(1000);
+                        })
+                        .catch(err => {
+                          setHasReport('not_started');
+                          setValidationError(err.message);
+                        });
+                    }}
+                  >
+                    Run Validation
+                  </Button>
+                )}
+              </Segment>
+            )}
+            {hasReport === 'running' && (
+              <Segment placeholder>
+                <Header icon>
+                  <Icon loading name="spinner" />
+                  Running data validation, this may take several minutes
+                </Header>
+                <Button
+                  negative
+                  onClick={() => {
+                    // setHasReport('canceled');
+                    cancelValidationRun({
+                      variables: {
+                        id: validationRun.id,
+                      },
+                    })
+                      .then(resp => {
+                        startPolling(1000);
+                      })
+                      .catch(err => {
+                        setHasReport('failed');
+                        setValidationError(err.message);
+                      });
+                  }}
+                >
+                  Cancel Validation
+                </Button>
+              </Segment>
+            )}
+            {hasReport === 'failed' && (
+              <Segment placeholder>
+                <Header icon>
+                  <Icon name="x" />
+                  Failed to run validation report.
```

Comment:
There is an issue with the validation running loader displaying in a certain scenario:

1. User creates review, adds a file to review that will fail validation (e.g. spend.docx)
2. User clicks Click to Start Validation button
3. Validation starts running
4. Validation fails and validation view shows the Failed to run validation report message with Run Again button
5. User removes bad file from review, then adds good file (e.g. A GWO manifest) that will complete validation
6. User navigates to validation view and clicks Run Again button
7. The view doesn't update at all. The running loader with the cancel validation button is expected

Reply:
I _think_ I'm seeing the same thing when I try to run a canceled run again by clicking Run Again button. The validation view doesn't change. I still see the Canceled message with the Run Again button.